modules/migration-installing-legacy-operator.adoc

```diff
@@ -43,18 +48,16 @@ $ podman login registry.redhat.io
 
 . Download the `operator.yml` file by entering the following command:
 +
-[source,terminal,subs="attributes+"]
+[source,terminal]
 ----
-$ podman cp $(podman create \
-  registry.redhat.io/rhmtc/openshift-migration-legacy-rhel8-operator:v{mtc-version}):/operator.yml ./
+podman cp $(podman create registry.redhat.io/rhmtc/openshift-migration-legacy-rhel8-operator:v1.7):/operator.yml ./
```

Comment:
I would recommend stating the commands are examples as some customers take them as verbatim and release versions might not meet their infrastructure configuration.

Reply:
I think this are meant to reflect the accurate commands.